tests/Fixer/ClassNotation/FinalInternalClassFixerTest.php

```diff
@@ -304,4 +306,77 @@ public function testConfigureSameAnnotationInBothLists(): void
             'annotation_exclude' => ['@internal123', 'b'],
         ]);
     }
+
+    /**
+     * @dataProvider provideAttributeExcludeCases
+     * @requires PHP 8.0
+     */
+    public function testAttributeExclude(string $expected, string $input, array $config): void
+    {
+        $config['consider_absent_docblock_as_internal_class'] = true;
+
+        $this->fixer->configure($config);
+        $this->doTest($expected, $input);
+    }
+
+    public function provideAttributeExcludeCases(): \Generator
```

Comment:
What happens if we run these tests on PHP 7?

Reply:
The attributes will be tokenized as comments so the fixer won't make changes. We could on > PHP8 already check comments and if these match the attribute-format fix accordingly, would it be worth it?

Reply:
maybe, but out of scope of this PR i guess :P